src/processor/tree/tree_wrapper.rs

```diff
@@ -186,19 +188,21 @@ fn reconstruct_genesis_state<D: Database>(
 
     tracing::trace!("Have {} unique keys in the tree", key_set.len());
 
-    let mut key_value_pairs: Vec<(U256, H256)> = Vec::with_capacity(batched.len());
+    let mut key_values = Vec::with_capacity(batched.len());
+    let mut index = 1;
```

Comment:
Instead of keeping track of index separately, can we enumerate the loop and use that value?

Reply:
well, we can, but we're already destructuring the items - do you really find
```
    for (index, (address, key, value)) in batched.iter().enumerate() {
```
more readable? Besides, the index will get both wrong type and wrong initial value by default...

Reply:
Ah, good points. I'll leave the decision up to you then